README.md

```
@@ -24,14 +24,16 @@ The most scalable open-source MQTT broker for IoT, IIoT, and connected vehicles.
 
 ## About
 
-[EMQX][emqx] is an Open-source MQTT broker with a high-performance real-time
-message processing engine, powering event streaming for IoT devices at massive
-scale. As the most scalable MQTT broker, EMQX can help you connect any device,
-at any scale (including your home).
-
-The [EMQX MQTT broker][emqx] is an advanced alternative to the Mosquitto MQTT
-broker/add-on that is generally used in Home Assistant. It has a UI
-to configure, manage, and debug your MQTT broker, clients, and traffic.
+[EMQX][https://www.emqx.com/en/products/emqx] EMQX is the world's most
```

markdownlint-cli2 (0.22.1)

[warning] Reference links and images should use a label that is defined
Missing link or image reference definition: "https://www.emqx.com/en/products/emqx"

(MD052, reference-links-images)